src/open_samus_returns_rando/files/custom/randoapi.lua

```diff
@@ -35,5 +35,5 @@ function RandoApi.CheckBeams()
     local hasPlasma = RandomizerPowerup.GetItemAmount("ITEM_WEAPON_PLASMA_BEAM") > 0
 
     -- Damage values are Solo Spazer, Solo Plasma, Plasma + Wave, and Plasma + Spazer, respectively
-    RandoApi.ChangeBeams(hasWave, hasSpazer, hasPlasma, 40, 85, 95, 90)
+    RandoApi.ChangeBeams(hasWave, hasSpazer, hasPlasma, 40, 60, 85, 75)
```

Comment:
Kinda funny that the first rebalance was from 75 to 85 for solo plasma because it felt to weak and now it is going down to 60.
Let's see how this feels now.